Memory/Hooks.cpp

```diff
@@ -976,6 +977,7 @@ void Hooks::Actor_lerpMotion(C_Entity* _this, vec3_t motVec) {
 
 		if (!networkSender) {
 			if (g_Data.getVersion() == GAMEVERSION::g_1_16_0)
+				// pretty sure its this Actor::lerpMotion
 				networkSender = reinterpret_cast<void*>(6 + FindSignature("FF 90 ?? ?? ?? ?? 4C 8D 9C 24 ?? ?? ?? ?? 49 8B 5B 18 49 8B 73 28 49 8B E3 5F C3"));
```

Comment:
I think this has to do with network instead of lerpMotion (the hook is lerpmotion)

Reply:
thanks, the devs can change it if they need toa